app/static/src/scss/includes/_search-transferring-body.scss

```diff
@@ -0,0 +1,218 @@
+  }
+}
+
+.govuk-grid-column-two-thirds {
+  &--search-results {
+    margin-top: 1rem;
+  }
+}
+
+.govuk-heading-m {
+  &--search-results {
+    font-size: 2rem;
+  }
+
+  &--search {
+    white-space: nowrap;
+    font-size: 1.3rem;
+    margin-bottom: 0.5rem;
+  }
+}
+
+@media screen and (width <=810px) {
+  .govuk-grid-row {
+    &--search {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+}
+
+.govuk-heading-s {
+  &--search-term {
+    margin-bottom: 0;
+    margin-top: 1rem;
+    padding-top: 0;
+    font-size: 1rem;
+  }
+}
+
+.govuk-table__cell {
+  &--search-results {
+    font-size: 1rem;
+    margin-bottom: -1rem;
+    line-height: 1.2rem;
+    padding: 10px;
+
+    &-no-wrap {
+      white-space: nowrap;
+    }
+
+    &:last-child {
+      padding-right: 0;
+    }
+
+    &:first-child {
+      padding-left: 0;
+    }
+
+    a {
+      color: #1d70b8;
+      margin-bottom: -1rem;
+
+      &:visited {
+        color: #1d70b8;
+      }
+    }
+  }
+}
+
+.govuk-table__header {
+  &--search-header {
+    font-size: 1rem;
+    padding: 10px;
+
+    &-title {
+      width: 100%;
+      font-size: 1rem;
+    }
+
+    &:last-child {
+      text-align: right;
+      padding-right: 0;
+    }
+
+    &:first-child {
+      padding-left: 0;
+    }
+  }
+}
+
+.search-all-filter-container {
+  background-color: #f3f2f1;
+  padding: 0.5rem;
+  padding-bottom: 1.5rem;
+  min-width: 100%;
+  margin-bottom: 0.5rem;
+  margin-top: 0.6rem;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+}
+
+.ayr-filter__tag {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.1429;
+  border: 1px solid #0b0c0c;
+  color: #0b0c0c;
+  display: inline-block;
+  margin-top: 5px;
+  padding: 5px;
+  text-decoration: none;
+  padding-right: 1.8rem;
+  height: 1.2rem;
+  padding-top: 10px;
+  width: 4rem;
+}
+
+.ayr-filter-tags {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding-top: 0.9rem;
+}
+
+img.cancel-filter {
+  fill: black;
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+
+  ::after {
+    content: "";
+    display: inline-block;
+    font-weight: bold;
+    height: 10px;
+    margin-left: 5px;
+    vertical-align: middle;
+    width: 10px;
+  }
+}
+
+a.govuk-link {
+  &--transferring-filter {
+    font-size: 0.9rem;
+
+    &:visited {
+      color: #1d70b8;
+    }
+  }
+}
+
+.govuk-button {
+  &__search-filters-form-apply-button {
+    height: 35px;
+    width: 70px;
+    font-size: 1rem;
+    margin: 0 26px 0 0;
+  }
+}
+
+.search-form__buttons {
+  display: flex;
+  align-items: center;
+  width: auto;
+  margin-top: 0.9rem;
+}
+
+.search-filter-link {
+  padding-top: 1rem;
+}
+
+.govuk-form-group {
+  &--search-all-filter {
+    margin-bottom: 0;
+  }
+
+  &__search-all-form-group {
+    display: inline-block;
+    margin-bottom: 0;
+  }
+}
+
+.govuk-input {
+  &--search-all-input {
+    width: 95% !important;
+  }
+}
+
+.box {
```

Comment:
is this following the css naming convention we agreed on @colinbowen ?

Reply:
.box can be renamed to something matching bem formatting.